config/routes.rb

```diff
@@ -40,4 +40,7 @@
   get '/applications/:id', to: 'applications#show'
   get '/pets/:pet_id/applications', to: 'application_pets#index'
   patch '/pets/:pet_id/:app_id', to: 'application_pets#update'
+  patch '/pets', to:'application_pets#batch_update'
```

Comment:
Maybe try to make this go to the pets controller since it is the exact resource we are updating. Try this during a refactor period.